### PR TITLE
feat(ONYX-1285): change of the image size dimensions of an artwork rail card

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [ONYX-1285] 

### Description

Implement a new Artwork Rail Card design (image only)
The artwork card within a rail should have 
- a consistent height of 215px
- a minimum width of 140px
- a maximum width of 340px
- If the image is thinner than the minimum width, a grey Black05 background is added. This should have a 10px padding at the top and bottom. 
- If the image is wider than the maximum width, a grey Black05 background is added. This should have a 10px padding on the left and right

| wide artwork | tall artwork | updated image skeleton New Home | updated image skeleton Old Home | outdated image skeleton Old Home |
| --- | --- | --- | --- | --- |
| Simulator Screenshot - iPhone 15 Plus - 2024-09-27 at 15 28 53 | Simulator Screenshot - iPhone 15 Plus - 2024-09-27 at 15 29 50 | Simulator Screenshot - iPhone 15 Plus - 2024-09-27 at 14 48 34 | Simulator Screenshot - iPhone 15 Plus - 2024-09-27 at 14 35 09 | Simulator Screenshot - iPhone 15 Plus - 2024-09-26 at 13 17 53 |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- change of the image size dimensions of an artwork rail card (behind a ff AREnableArtworkRailRedesignImageAspectRatio)- daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1285]: https://artsyproduct.atlassian.net/browse/ONYX-1285?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (change of the image size dimensions of an artwork rail card (behind a ff AREnableArtworkRailRedesignImageAspectRatio)- daria)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 25da38487a53ee60cd25b362895715314147d05c
</p>

**Review by anandaroop — APPROVED** [review]
Looks great!

I'll add a screencap of a few artists from the [aspect ratio stress test](https://observablehq.com/@anandaroop/aspect-ratio-stress-test):

File: src/app/Components/ArtworkRail/ArtworkRail.tsx [hunk 1]
```diff
@@ -23,7 +28,7 @@ import { isTablet } from "react-native-device-info"
 import { graphql, useFragment } from "react-relay"
 
 export const INITIAL_NUM_TO_RENDER = isTablet() ? 10 : 5
-export const ARTWORK_RAIL_IMAGE_WIDTH = 295
+export const ARTWORK_RAIL_IMAGE_WIDTH = 295 // here
```
[the remark this answers is not in the document]
Comment (dariakoko) [reply src/app/Components/ArtworkRail/ArtworkRail.tsx:31]: Nope, will remove :)

**Comment by dariakoko** [comment]
Thanks for the recording @anandaroop!
I also, added a handling of the case when aspect ration is not available and added tests

**Review by anandaroop — APPROVED** [review]

File: src/app/Components/ArtworkRail/ArtworkRail.tsx [hunk 2]
```diff
@@ -133,15 +138,31 @@ const artworksFragment = graphql`
...
+              height={ARTWORK_RAIL_CARD_IMAGE_HEIGHT}
+              width={ARTWORK_RAIL_MIN_IMAGE_WIDTH * 2}
+            />
+          ) : (
+            <SkeletonBox
+              height={LEGACY_ARTWORK_RAIL_CARD_IMAGE_HEIGHT}
+              width={ARTWORK_RAIL_IMAGE_WIDTH}
+            />
+          )}
+          <Spacer y={2} />
```
Comment (araujobarret) [inline src/app/Components/ArtworkRail/ArtworkRail.tsx:161]: Spacer is not needed here, we should have used `gap` property on line 149

**Pull request merged**